doc/REST-interface.md

```diff
@@ -31,6 +31,11 @@ With the /notxdetails/ option JSON response will only contain the transaction ha
 
 Given a block hash: returns <COUNT> amount of blockheaders in upward direction.
 
+#### Blockhash by height
+`GET /rest/blockhashbyheight/<HEIGHT>.<bin|hex|json>`
+
+Given a height: returns blockhash of block in main chain at height `<HEIGHT>`.
```

Comment:
The help for RPC `getblockhash` is `Returns hash of block in best-block-chain at height provided.` I think that's slightly better terminology to use here than "main chain" which could be confused with testnet/mainnet/....